ESCPOS_NET/Emitters/Enums/Color.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace ESCPOS_NET.Emitters
+{
+    [Flags]
+    [System.Diagnostics.CodeAnalysis.SuppressMessage("StyleCop.CSharp.SpacingRules", "SA1025:Code should not contain multiple whitespace in a row", Justification = "Enums are easier to read if they have whitespace alignment.")]
```

Comment:
I agree with the justification, but the enum below doesn't have whitespace alignment. Can you move this to the suppressions file instead?